mdpow/tests/test_solv_shell.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+from . import tempdir as td
+
+import py.path
+
+import pybol
+import pytest
+
+from numpy.testing import assert_almost_equal
+from scipy.stats import variation
```

Comment:
My view is that this is too elaborate, keep it simple and just use numpy. I had to look up what `scipy.stats.variation` does, which was a sign that too much is going on than necessary.

Reply:
remove